docs/source/package_reference/hf_api.mdx

```diff
@@ -2,46 +2,53 @@
 
 [[autodoc]] login
```

Comment:
This should be removed, it's deprecated.

Reply:
I've manually added deprecation warnings with the expected version removal; as those methods are points of entry I fear removing the methods themselves would lead to users Ctrl+F and not finding what they're looking for. I've added a redirection to the methods that should now be used instead.

Let me know if that works for you.